### ISSUE TITLE
Support for Confluent Schema Registry?

### ISSUE THREAD

**Issue by DaveSlinn** [fluent/fluent-plugin-kafka#288]
We are using https://github.com/takebayashi/fluent-plugin-avro to format our messages as Avro, but when sending to our kafka cluster, we need to prefix the avro data with 5 bytes to conform with the Confluent Schema Registry, as described here:

 https://docs.confluent.io/current/schema-registry/serializer-formatter.html#wire-format

Is this possible?

**Comment by rverma-jm** [comment]
@DaveSlinn did you find any solution for your usecase?

**Comment by DaveSlinn** [comment]
Yes, a co-worker of mine forked the Avro plugin and added the ability to specify the confluent schema id.  His forked repo is here: https://github.com/mstinglin/fluent-plugin-avro.  We've been using it for months now.  Works great.
